src/System.Net.Sockets/src/System/Net/Sockets/SafeSocketHandle.Windows.cs

```diff
@@ -108,14 +108,16 @@ internal static SafeSocketHandle CreateWSASocket(AddressFamily addressFamily, So
             return CreateSocket(InnerSafeCloseSocket.Accept(socketHandle, socketAddress, ref socketAddressSize));
         }
 
-        private void InnerReleaseHandle()
+        private bool InnerReleaseHandle()
```

Comment:
why do we change this is we never return true?

Reply:
In common code between Unix and Windows we use the return value.